src/pages/components/footer.mdx

```diff
@@ -11,62 +11,243 @@ import ComponentFeedback from "components/ComponentFeedback";
+## Variations
 
-![Image of minimal footer](../../images/component/footer/footer-demo-minimal.jpg)
 
-<br />
+### Short
 
-## Micro Footer
+Short footer removes all the high-level corporate navigation, leaving the required legal links, the optional adjunct legal links, the locale selector and branding.
 
-This version only shows the legal links, which are required, and the optional locale selector to maximize the space for the content of the given page. This footer is reserved for stand-alone experiences such as product demos, or slide shows. All other pages and experiences on IBM.com should use the default or the short versions.
+![Image of minimal footer](../../images/component/footer/footer_short_01.jpg)
 
-![Image of micro footer](../../images/component/footer/footer-demo-micro.jpg)
+<Caption>Short footer at xlg 1312px breakpoint</Caption>
 
+#### When to use
+Short footer is reserved for approved transactional pages, such as check out and registration experiences.
 <br />
 
-## Design and functional specifications
 
-For more details, [see design and functional specs](https://ibm.box.com/s/nsbccv6jigfw78uwq3r0242qks1x1f87).
+### Micro
+
+Micro footer variation only shows the required legal links and the optional locale selector to maximize the space for the page content. Adjunct legal links are not allowed. If you need them, please use either the Default, or the Short footer.
+
+![Image of micro footer](../../images/component/footer/footer_micro_01.jpg)
+
+<Caption>Micro Footer at xlg 1312px breakpoint</Caption>
+
+
+#### When to use
+This footer is reserved for stand-alone experiences such as product demos, or slide shows. All other pages and experiences on IBM.com should use the Default or the Short variations.<br />
+
+
+
+
+### With adjunct legal links
+
+Only the Default and the Short footers may contain up to 2 adjunct legal links for legally required notes or messages. They are displayed at the very bottom of the page, under the main legal links.
+
+![Image of micro footer](../../images/component/footer/footer_default_withAdjunctlegallinks_02.jpg)
+
+<Caption>Default footer at max 1584px breakpoint with one optional adjunct legal link</Caption>
 
 <br />
 
-## Development documentation
+1. **Adjunct legal link:** Use up to two adjunct legal links regarding changes, policies or trademark issues.
+1. **Legal links:** The general Legal links are present on every IBM footer and cannot be changed.
 
-<ComponentStatus name="Footer" type="ui" />
+<br />
+<br />
+
+![Image of micro footer](../../images/component/footer/footer_default_02.jpg)
+
+<Caption>Short Footer at xlg 1312px breakpoint, with one optional adjunct legal link</Caption>
 
 <br />
 
+#### When to use
+Use up to two adjunct legal links for legally required notes or messages legally required to provide, regarding changes, policies or trademark issues.
+<br>
```

Comment:
@DragosRistici The deploy preview is failing because `<br>` must be closed:

```suggestion
<br />
```

Reply:
thx